bittensor/_neuron/text/neuron_utilities.py

```diff
@@ -1,8 +1,33 @@
+from numpy import zeros_like
 import bittensor
 import torch
+import torch.nn as nn
 import torch.nn.functional as F
 
-
+def update_metagraph_peerweight(metagraph, nucleus, device):
+    r"""
+    Check for the change in hotkey before and after metagraph sync, 
+    update the peer_weight of nucleus accordingingly.
+        Args:
+            metagraph (:obj:`bittensor.metagraph`, `required`):
+                The metagraph to sync.
+            nucleus (:obj:`bittensor.neuron.text.nucleus`, `required`):
+                The nn.Module class that needs the peerweight to be updated.
+            device (:type:`torch.device`)
+                The device where peer_weight should be stored. 
+    """ 
+    old_hotkeys = metagraph.hotkeys
+    metagraph.sync()
```

Comment:
This will sync the metagraph in the neuron as well, correct?

Reply:
yea, cause they are the same